conf/scylla.yaml

```diff
@@ -510,3 +510,20 @@ commitlog_total_space_in_mb: -1
 #
 # Keep at 12 for new clusters.
 murmur3_partitioner_ignore_msb_bits: 12
+
+# Use a new implementation of reversed reads in sstables when performing
+# reversed queries. Bypasses the cache for reversed reads even when
+# `bypass cache` is not explicitly specified in the query.
+# The new implementation does not require unbounded memory
+# (compared to the old implementation which had to fetch entire partitions
+# into memory) but does not work with the cache yet. Turn this option off
+# if your partitions are small so the old implementation is good enough;
+# your queries can then utilize the cache and potentially be faster
+# (since they don't need to use sstables as much). You can also turn
+# this option off and explicitly perform `bypass cache` queries to
+# use the new method.
+# This option is temporary and will be removed as soon as the cache
+# and sstable reverse read algorithms are updated to handle reversed
+# queries correctly.
+# The option can be updated without restarting Scylla.
+# reversed_reads_auto_bypass_cache: true
```

Comment:
> "do not use reverse readers" (at all)

What would that do? What would happen if the user requested a reversed read from CQL? Error?

I thought the intent of this option was for the user to be able to use the new efficient reversing code seamlessly in their applications, which requires bypassing the cache at this point. So in order for them to not have to insert `bypass cache` everywhere in their queries (and then in the future remove it) I added an option to do it automatically.

Reply:
If reverse readers are disabled by users, the request would be executed using the old logic, by querying in forward mode and reversing in memory. 

I thought the reason for this option was that this performs better than using reversed readers if the partition is small. Another reason could be to workaround bugs in the new reversed readers. 

Maybe a global option for bypassing cache for reversed readers also makes sense, even if cache supports reverse reads. I do hope that we will manage to release reversed reader support in cache together with everything. But in case of bugs it may be good to have it.